src/components/Layout/NavigationBar/NavigationBar.tsx

```diff
@@ -1,203 +1,164 @@
 import styled from "@emotion/styled";
-import { Dropdown, DropdownItem } from "@govtechsg/tradetrust-ui-components";
-import React, { useState } from "react";
+import React, { FunctionComponent } from "react";
 import { Settings } from "react-feather";
-import { useLocation } from "react-router-dom";
-import { NavHashLink } from "react-router-hash-link";
-import tw from "twin.macro";
+import { NavigationBar as NavBar } from "@govtechsg/tradetrust-ui-components";
+import { NavigationItem, NavigationItemType } from "./type";
+import { NavigationBarItem } from "./NavigationBarItem";
 
-interface NavItemsProps {
-  id: string;
-  label: string;
-  path: string;
-  dropdownItems?: { id: string; label: string; path: string }[];
-}
+const NavigationBarStyle = styled.div`
+  .create-btn {
```

Comment:
wanna use the exported on from ui ?

Reply:
I tried, seems like there is error when change pass across node_module